data-prepper-plugins/common/src/test/java/com/amazon/dataprepper/plugins/prepper/StringPrepperTests.java

```diff
@@ -0,0 +1,55 @@
+package com.amazon.dataprepper.plugins.prepper;
+
+import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.amazon.dataprepper.model.record.Record;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class StringPrepperTests {
+
+    private final String PLUGIN_NAME = "string_converter";
+
+    private final String TEST_DATA_1 = "data_prepper";
+    private final String TEST_DATA_2 = "STRING_CONVERTER";
```

Comment:
nit - might name these something like UPPERCASE_TEST_STRING and LOWERCASE_TEST_STRING to make it more obvious that they have special properties (i.e. they aren't just random strings).